keg_elements/forms/__init__.py

```diff
@@ -200,6 +201,26 @@ def selected_choice_label(self):
         return value_dict.get(self.data)
 
 
+class RequiredRadioBooleanField(wtforms.fields.RadioField):
+    def __init__(self, *args, **kwargs):
+        yes_label = kwargs.pop('yes_label', 'Yes')
+        no_label = kwargs.pop('no_label', 'No')
+
+        def bool_coerce(val):
+            if val == u'True':
+                return True
+            if val == u'False':
+                return False
+            return val
+
+        kwargs['choices'] = [(True, yes_label), (False, no_label)]
+        kwargs['coerce'] = bool_coerce
+        kwargs['validators'] = [InputRequired()]
```

Comment:
I'll add support for more validators. I need to think through the implications of supporting different semantic boolean types.